src/traction_modem/RxFlow.hxx

```diff
@@ -83,9 +83,18 @@ public:
     {
         return resyncCount_;
     }
+
+    /// Register for a callback when any packet is received.
+    virtual void register_packet_rx_callback(std::function<void()> callback)
```

Comment:
this probably should not be virtual, since the only meaningful implementation is inline here.